src/packages/solid/components/Row/Row.tsx

```diff
@@ -15,7 +15,6 @@ const Row: Component<RowProps> = (props: RowProps) => {
       {...props}
       animate
       style={styles.Container}
-      clipping
       onSelectedChanged={withScrolling(props.x as number)}
```

Comment:
Also need to support chaining onSelectedChanged so users can pass in another function